UNITTESTS/fakes/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+# add all subdirectories
+
+file(GLOB subdirectories RELATIVE ${CMAKE_CURRENT_LIST_DIR} *)
```

Comment:
do we need globing ?

Reply:
I think file with GLOB lists directories and we use the list to iterate to add_subdirectory

Reply:
CMake doesn't recommend globbing https://cmake.org/cmake/help/latest/command/file.html#filesystem

Reply:
I know but I think they mean for files only, this is just for add_subdirectory